src/displayapp/screens/WatchFaceFuturistic.cpp

```diff
@@ -0,0 +1,185 @@
+
+  arc_hour = lv_arc_create(lv_scr_act(), nullptr);
+  lv_obj_set_size(arc_hour, 140, 140);
+  lv_arc_set_range(arc_hour, 0, 360);
+  lv_obj_align(arc_hour, lv_scr_act(), LV_ALIGN_CENTER, 0, 0);
+  lv_arc_set_rotation(arc_hour, 270);
+  lv_arc_set_bg_angles(arc_hour, 0, 360);
+  lv_arc_set_adjustable(arc_hour, false);
+  lv_arc_set_start_angle(arc_hour, 0);
+  lv_obj_add_style(arc_hour, LV_ARC_PART_INDIC, &arc_fg_style);
+  lv_obj_add_style(arc_hour, LV_ARC_PART_BG, &arc_bg_style);
+  lv_obj_set_style_local_line_color(arc_hour, LV_ARC_PART_INDIC, LV_STATE_DEFAULT, lv_color_hex(hoursArcColor));
+
+  label = lv_label_create(lv_scr_act(), nullptr);
+  lv_obj_set_style_local_text_color(label, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, lv_color_hex(innerTextColor));
+  lv_label_set_text_fmt(label, "[init]");
+  lv_label_set_align(label, LV_LABEL_ALIGN_CENTER);
+  lv_obj_align(label, nullptr, LV_ALIGN_CENTER, 0, -12);
+
+  taskRefresh = lv_task_create(RefreshTaskCallback, LV_DISP_DEF_REFR_PERIOD, LV_TASK_PRIO_MID, this);
+
+  Refresh();
+}
+
+WatchFaceFuturistic::~WatchFaceFuturistic() {
+  lv_task_del(taskRefresh);
+
+  lv_style_reset(&arc_fg_style);
+  lv_style_reset(&arc_bg_style);
+
+  lv_obj_clean(lv_scr_act());
+}
+
+void WatchFaceFuturistic::Refresh() {
+  heartbeat = heartRateController.HeartRate();
+  heartbeatRunning = heartRateController.State() != Controllers::HeartRateController::States::Stopped;
+  if (heartbeat.IsUpdated() || heartbeatRunning.IsUpdated()) {
+    if (heartbeatRunning.Get()) {
+      lv_obj_set_hidden(heartbeatIcon, false);
+      lv_obj_set_hidden(heartbeatValue, false);
+      lv_label_set_text_fmt(heartbeatValue, "%03d", heartbeat.Get());
+    } else {
+      lv_obj_set_hidden(heartbeatIcon, true);
+      lv_obj_set_hidden(heartbeatValue, true);
+    }
+
+    lv_obj_realign(heartbeatIcon);
+    lv_obj_realign(heartbeatValue);
+  }
+
+  batteryPercentRemaining = batteryController.PercentRemaining();
+  isCharging = batteryController.IsCharging();
+  if (isCharging.IsUpdated() || batteryPercentRemaining.IsUpdated()) {
+    if (isCharging.Get()) {
+      lv_obj_set_hidden(plugIcon, false);
+      lv_obj_set_hidden(batteryStatus, false);
+      lv_label_set_text_fmt(batteryStatus, "%03d%%", batteryPercentRemaining.Get());
+      lv_obj_set_hidden(batteryAlert, true);
+    } else {
+      lv_obj_set_hidden(plugIcon, true);
+      lv_obj_set_hidden(batteryStatus, true);
+
+      if (batteryPercentRemaining.Get() < 15) {
+        lv_obj_set_hidden(batteryAlert, false);
+      } else {
+        lv_obj_set_hidden(batteryAlert, true);
+      }
+    }
+  }
+
+  currentDateTime = dateTimeController.CurrentDateTime();
+  if (currentDateTime.IsUpdated()) {
+    uint8_t day = dateTimeController.Day();
+    uint8_t hour = dateTimeController.Hours();
+    uint8_t minute = dateTimeController.Minutes();
+    uint8_t second = dateTimeController.Seconds();
+
+    if (sSecond != second) {
+      lv_arc_set_value(arc_sec, second * (360 / 60));
```

Comment:
You can modify the arc ranges, so you can pass the time without modification.
```suggestion
      lv_arc_set_value(arc_sec, second);
```